frontend/apps/filemanager/filemanager.lua

```diff
@@ -157,6 +157,30 @@ function FileManager:init()
                         UIManager:close(self.file_dialog)
                     end,
                 },
+                {
+                    text = _("Purge .sdr"),
+                    enabled = lfs.attributes(util.realpath(file), "mode") == "file" and util.realpath(file):match("%.") and
+                            lfs.attributes(util.realpath(file):match("(.*)%.")..".sdr", "mode") == "directory",
+                    callback = function()
+                        local purgeDir = require("ffi/util").purgeDir
+                        local full_path = util.realpath(file)
+                        local sidecar = full_path:match("%.") and full_path:match("(.*)%.")..".sdr" or nil
```

Comment:
Better use getSidecarDir to get the sidecard directory path. You can find examples within this file.